# Ok Cupid Dataset EDA

This dataset was picked on the name alone to do an initial clean and EDA.
The dataset is completly unfamiliar to me - I want to practice familiarising myself with the set and seeing what I can learn about it, to see what further investigation can stem from it. 

In [6]:
# initial imports and check file is readable
import pandas as pd
okcupid = pd.read_csv('OkCupid_kaggle_dataset.zip')


In [7]:
print(okcupid.head((10)))

   age     status sex orientation       body_type               diet  \
0   22     single   m    straight  a little extra  strictly anything   
1   35     single   m    straight         average       mostly other   
2   38  available   m    straight            thin           anything   
3   23     single   m    straight            thin         vegetarian   
4   29     single   m    straight        athletic                NaN   
5   29     single   m    straight         average    mostly anything   
6   32     single   f    straight             fit  strictly anything   
7   31     single   f    straight         average    mostly anything   
8   24     single   f    straight             NaN  strictly anything   
9   37     single   m    straight        athletic    mostly anything   

       drinks      drugs                          education  \
0    socially      never      working on college/university   
1       often  sometimes              working on space camp   
2    socially     

In [2]:
okcupid.shape

(59946, 31)

This set has 59946 rows and 31 columns

## Duplicates check

In [11]:
duplicates = okcupid.duplicated()
print(duplicates.value_counts())

False    59946
dtype: int64


The number of false duplicates equals the number of rows - i.e. there are no dupicated rows